src/app/routes/regex/index.js

```diff
@@ -18,4 +18,8 @@ export const frontpageRegexPath = `/:service(${serviceRegex}):amp(${ampRegex})?`
 
 export const frontpageDataRegexPath = `${frontpageRegexPath}.json`;
 
+export const manifestRegexPath = `/:service(${serviceRegex})/articles/manifest.json`;
```

Comment:
This is a duplicate of `articleManifestRegexPath` above

Reply:
@david-boydell Need some help with this